Simplify internal modules.

jtobin opened the pull request:
Just some simple cleaning I did after somehow looking at the various `internal` modules.  Chops out a bunch of unnecessary/redundant code there.

jtobin committed:
Clean up internal modules.

Remove superfluous return statements in basic internal wrappers.

jtobin committed:
DRY in Ecliptic transaction generation.

The internal/ecliptic.js module contained a lot of repetition (with a
comment explaining it); this just follows that comment's advice and
employs a general transaction-generation pattern.

Fang- commented on internal/ecliptic.js:
```diff
@@ -1,238 +1,113 @@
-               _from, _to, _point);
-  let abi  = data.encodeABI();
-  return tx(addr, abi, 0);
+function txFromMethod(contracts, method, ...args) {
```
Why not put this to use in the other method files also? You could probably throw this into a helper lib, constructing this function by passing in a `contracts.whatever` object.

jtobin replied:
« Bonne idée » !

Fang- reviewed (commented):
Not sure what removing """superfluous""" syntax gets us. afaic most of JS's sugar just makes code take more effort to follow. Removing semicolons is the worst part here.

But eh, I'm not the one building releases.

jtobin committed:
Add internal/genTransaction module.

Simply provides a single 'tx' function for generating unsigned
transactions for specified contract methods.

jtobin committed:
Simplify gen.tx a little.
